HW02/Pavel_Karenda/pavel_karenda_hw_02_t_01.rb

```diff
@@ -0,0 +1,30 @@
+ERROR = /error/.freeze
+arr_lines = []
+
+if ARGV.length != 1
+  puts 'We need exactly one parameter. The name of a file.'
+  exit
+end
+filename = ARGV[0]
+puts "Going to open '#{filename}'"
+
+file = File.open(filename)
+
+def find_error(file, arr_lines)
+  file.each do |line|
+    arr_lines.push(line) if line.match(ERROR)
+  end
+  arr_lines
+end
+p lines = find_error(file, arr_lines)
```

Comment:
You pass an empty array to the method, can it be better to declare it there or assign a return value to it?
```suggestion
def find_error(file)
  arr_lines = []
  file.each do |line|
    arr_lines.push(line) if line.match(ERROR)
  end
  arr_lines
end
p lines = find_error(file)
```

If you do so, then do not forget to delete the "arr_lines = []" at the beginning of the file